docs/output-logstash.asciidoc

```diff
@@ -213,66 +193,83 @@ This setting can be used to _override_ the system trust store for verifying the
-
-Password for the <<plugins-{type}s-{plugin}-ssl_keystore_path>>
-
 [id="plugins-{type}s-{plugin}-ssl_keystore_path"]
 ===== `ssl_keystore_path`
 
 * Value type is <<path,path>>
 * There is no default value for this setting.
 * When present, <<plugins-{type}s-{plugin}-ssl_keystore_password>> is also required.
-* {prohibit-ssl-disabled-effective}
+* Cannot be combined with configurations that disable SSL.
 
 A path to a JKS- or PKCS12-formatted keystore with which to identify this plugin to the downstream input.
 The provided identity will be used if the downstream input enables <<plugins-{type}s-{plugin}-config-ssl-trust,SSL client authentication>>.
 
-[id="plugins-{type}s-{plugin}-ssl_truststore_password"]
-===== `ssl_truststore_password`
+[id="plugins-{type}s-{plugin}-ssl_keystore_password"]
+===== `ssl_keystore_password`
 
 * Value type is <<password,password>>
 * There is no default value for this setting.
-* Required when connection identity is configured with <<plugins-{type}s-{plugin}-ssl_truststore_path>>
-* {prohibit-ssl-disabled-effective}
+* Required when connection identity is configured with <<plugins-{type}s-{plugin}-ssl_keystore_path>>
+* Cannot be combined with configurations that disable SSL.
 
-Password for the <<plugins-{type}s-{plugin}-ssl_truststore_path>>
+Password for the <<plugins-{type}s-{plugin}-ssl_keystore_path>>
 
 [id="plugins-{type}s-{plugin}-ssl_truststore_path"]
 ===== `ssl_truststore_path`
 
 * Value type is <<path,path>>
 * There is no default value for this setting.
 * When present, <<plugins-{type}s-{plugin}-ssl_truststore_path>> is also required.
-* {prohibit-ssl-disabled-effective}
-* {prohibit-ssl-verification-mode-none}
+* Cannot be combined with configurations that disable SSL.
+* Cannot be combined with <<plugins-{type}s-{plugin}-ssl_verification_mode, `+ssl_verification_mode => none+`>>.
 
 A path to a JKS- or PKCS12-formatted truststore with which to validate the identity claims of the downstream input.
 The provided identity will be used if the downstream input enables <<plugins-{type}s-{plugin}-config-ssl-trust,SSL client authentication>>.
 
+[id="plugins-{type}s-{plugin}-ssl_truststore_password"]
+===== `ssl_truststore_password`
+
+* Value type is <<password,password>>
+* There is no default value for this setting.
+* Required when connection identity is configured with <<plugins-{type}s-{plugin}-ssl_truststore_path>>
+* Cannot be combined with configurations that disable SSL.
+
+Password for the <<plugins-{type}s-{plugin}-ssl_truststore_path>>
 
 [id="plugins-{type}s-{plugin}-ssl_verification_mode"]
 ===== `ssl_verification_mode`
 
 * Value type is <<string,string>>
-* There is only one currently-supported mode:
+* The supported modes are:
 ** `full`: verifies that a certificate provided by the client has an identity claim matching <<plugins-{type}s-{plugin}-host>>, is signed by a trusted authority (CA), is within its valid date range, and that the client has possession of the associated key.
```

Comment:
Rephrase: the output plugin uses `ssl_verification_mode` for validating a downstream input, not a [connecting] client.

```suggestion
** `full`: verifies that a certificate provided by the downstream input has an identity claim matching <<plugins-{type}s-{plugin}-host>>, is signed by a trusted authority (CA), is within its valid date range, and that the downstream input has possession of the associated key.
```